### DISCUSSION TITLE
built in marker supprt and marker icon reszizing

### DISCUSSION THREAD

**Discussion by BorealBlizzard** [NeumimTo/Pl3xMap#11] [Ideas]
Could we get some built in marker commands? commands that work similar to dynmap commands would be sweet. 
It would also be nice to have a size option in the config, maybe its already there and i missed it.

**Comment by NeumimTo** [comment]
https://www.spigotmc.org/resources/pl3xmap-marker.97262/ Have you tried this resource?

**Reply by BorealBlizzard** [reply]
yes i have, doesnt really have the use of usage i was looking for, mainly the fact that each marker has to have a numeric id that increments

**Reply by SentixDev** [reply]
Actually, that sounds like a cool idea for next update, to determine on marker creation automatically an ID when there's no ID given. Gonna implement that in the next days/week. :)

**Reply by SentixDev** [reply]
https://github.com/SentixDev/Pl3xMap-Marker/releases
